app/models/blog.rb

```diff
@@ -0,0 +1,19 @@
+class Blog < ApplicationRecord
+	belongs_to :user
+	has_rich_text :description
+
+	validates :title, :presence => true, :length => { :minimum => 10 }, :format => { :with => /\A[a-zA-Z0-9\s]+\z/, :message => "can only contain letters, numbers, and spaces" }
+	validate :description_has_at_least_one_line
+
+	private
+
+	def description_has_at_least_one_line
+		if description.blank? || description.body.to_plain_text.strip.split("\n").all?(&:blank?)
+			errors.add(:description, "must contain at least one non-blank line")
+		end
+	end
+
+	def self.create(title, description, user_id)
+		self.new({:title => title, :description => description, :user_id => user_id})
+	end
```

Comment:
naming is not correct
when you say create it means it will create like you do for user
but in this you are only initializing

so you may use get_new or initialize but not create . i you want then save it here